iothub/device/devdoc/Convention-based operations.md

```diff
@@ -6,6 +6,14 @@
 public class ClientOptions {
 +    public PayloadConvention PayloadConvention { get; set; }
```

Comment:
probably should be readonly too

Reply:
So, in reality, you can modify these options as many times as you want until you actually pass it to the client, so I don't think a setter is entirely misleading here. 
I added the following text to the doc comments instead, to highlight this behavior:
"Updating these options after the client has been initialized will not change the behavior of the client."

What do you think?

Reply:
It also means they could pass it to the client, and change it after. We're both looking at the same instance.

Reply:
Synced up offline, I'll make the change.